src/ethereum/info/ftminfo.js

```diff
@@ -49,6 +49,7 @@ const otherSettings: EthereumSettings = {
     name: 'Fantom Opera'
   },
   hdPathCoinType: 60,
+  supportsEIP1559: true,
```

Comment:
Source? I looked for this and couldn't find an announcement for fantom specifically